### ISSUE TITLE
The Project is actually getting worse due to conflicting merging of PRs.

### ISSUE THREAD

**Issue by VanshGarg06** [iamrahulmahato/master-web-development#2232]
The Project is actually getting worse as the beginning of the project was good but after the of October things are getting worse then i think i have been working on this repository since a month and in the beginning it was actually great but now something are not working well and even the alignment of elements gets disturbed with wrong merging of the major conflicting PRs.

I would like to request the maintainer that please take a note that what PR is actually doing and i know that it would be difficult to dedicate much time to a particular PR. But trust me things are actually getting worse. I myself have resolved issue of Footer 7 times and cloned the project several times. 

So , i would like to request all the contributors and maintainer that please take due care of the project as it was a nice project but now a lot of blunders are happening. 

One of the blunder i have seen that with footer i have been working on it and someone actually hide the Subscribe button just by applying z-index property which is actually not a good practice and such contributions must not be appreciated.

Contribute to learn not for the sake of just creating more PRs and getting score for GSSOC_EXT 24.

**Comment by iamrahulmahato** [comment]
@VanshGarg06 Yes, you are right; it is really challenging to read the minor codes, but I am trying my hardest. The problem is that there are only two days left and a lot of PR requests are approaching.
